### ISSUE TITLE
Browsing with tabulation in Carousel can switch on hidden cards with buttons

### ISSUE THREAD

**Issue by correi-f** [theopenconversationkit/tock-react-kit#102]
Buttons on hidden cards of carousel can be accessed when an user tab. This is a problem for accessibility, only visible cards of the Carousel should be accessed by keyboard browsing.

![image](https://user-images.githubusercontent.com/2005914/174995927-d8fc78ef-e8d6-4f78-bf41-30e1c258adb9.png)

Possible solution : Buttons on hidden cards must have tabIndex="-1"  as attribute.
